packages/frontend/src/components/Search/FilterSearch.tsx

```diff
@@ -127,6 +143,29 @@ export const FilterSearch = () => {
         </Button>
       </Toolbar>
       <Box>
+        <ListItem>
+          <ListItemText disableTypography>
+            <Typography variant="body1" textTransform="uppercase" sx={{ color: Colors.darkgrey }}>
+              {t('search:sort_results')}
+            </Typography>
+          </ListItemText>
+        </ListItem>
+        <ListItem>
+          <Select
+            displayEmpty
+            label={t('search:sort_results')}
+            value={sortType}
+            onChange={(event) => setSortType(event.target.value)}
+            input={<OutlinedInput />}
+            sx={{ width: '100%' }}
+          >
+            <MenuItem value={SortDataType.NONE}>{t('search:no_sort')}</MenuItem>
+            <MenuItem value={SortDataType.ASC_NAME}>A–Z</MenuItem>
+            <MenuItem value={SortDataType.DESC_NAME}>Z-A</MenuItem>
```

Comment:
Finnish alphabet goes from A to Ö. Though it's generally understood what A-Z ordering means. 😄

Reply:
Good notice. I'll add to the translation file